### PR TITLE
Fixing Xaml bug on 15.4 branch

### PR DESCRIPTION
**Pull request by DavidKarlas**
Also needs https://github.com/xamarin/md-addins/pull/2379 to be merged

### CODE REVIEW

**Comment by slluis** [comment]
@Aguilex @jmatthiesen @AdrianMurphy can somebody please merge? QA accepted.

File: version-checks [hunk 1]
```diff
@@ -17,8 +17,8 @@ DEP[0]=md-addins
 DEP_NAME[0]=MDADDINS
 DEP_PATH[0]=${top_srcdir}/../md-addins
 DEP_MODULE[0]=git@github.com:xamarin/md-addins.git
-DEP_NEEDED_VERSION[0]=653cdc8f0fb5b11be88a29342adc66156d50fd9c
-DEP_BRANCH_AND_REMOTE[0]="d15-4 origin/d15-4"
+DEP_NEEDED_VERSION[0]=7ee7b94a4fe57f06c8d258a0aca743cf6f1063b3
+DEP_BRANCH_AND_REMOTE[0]="d15-4Xaml origin/d15-4Xaml"
```
Comment (alanmcgovern) [inline version-checks]: This should probably not be merged into master. We need to keep the original `d15-4` branch

Comment (DavidKarlas) [reply version-checks]: thats what is causing conflict... I did this to get wrench build for QA, will fix in minutes

Comment (alanmcgovern) [reply version-checks]: :+1:

**Pull request merged**